library/src/main/java/com/trendyol/showcase/ui/tooltip/TooltipViewState.kt

```diff
@@ -15,9 +15,21 @@ data class TooltipViewState(
     val arrowPercentage: Int?,
```

Comment:
Dear Belfu, can we add unit tests for  TooltipViewState.kt if it's not already added